src/wallet/components/Wallet.tsx

```diff
@@ -51,23 +68,31 @@ function WalletContent({
     isConnectModalOpen,
     handleClose,
     connectRef,
-    showSubComponentAbove,
-    alignSubComponentRight,
     breakpoint,
   } = useWalletContext();
   const walletContainerRef = useRef<HTMLDivElement>(null);
 
   useOutsideClick(walletContainerRef, handleClose);
 
-  const { connect, dropdown, advanced } = useMemo(() => {
+  const { dropdown, advanced } = useMemo(() => {
     const childrenArray = Children.toArray(children);
     return {
-      connect: childrenArray.find(findComponent(ConnectWallet)),
       dropdown: childrenArray.find(findComponent(WalletDropdown)),
       advanced: childrenArray.find(findComponent(WalletAdvanced)),
     };
   }, [children]);
 
+  // cannot use advanced and dropdown,
+  // default to dropdown
+  const childrenToRender = useMemo(() => {
+    return Children.map(children, (child: ReactNode) => {
+      if (isValidElement(child) && child.type === WalletAdvanced && dropdown) {
+        return null;
```

Comment:
suggestion(non-blocking): Not sure if this would be a breaking change and we don't need to address it since a lot of this is in flux, but as a consumer, I would rather an invalid state throw a descriptive error in this case. It'd be easier to understand why something isn't working as expected

Reply:
Another suggestion would be to just return `defaultWalletChildren` here and not do the `||` check on line 118.